python/tutorials/01-vector-add.py

```diff
@@ -137,3 +139,138 @@ def benchmark(size, provider):
 # We can now run the decorated function above. Pass `print_data=True` to see the performance number, `show_plots=True` to plot them, and/or
 # `save_path='/path/to/results/' to save them to disk along with raw CSV data:
 benchmark.run(print_data=True, show_plots=True)
+
+
+# %%
+# We can also vary the number of arguments to see the power of kernel fusion.
+# We will create a fused kernel function to replace the pytorch style code:
+# `tensor1 + tensor2 + ... + tensorn`.
+# In fact, `triton` gets its name because triton (the hydrogen atom with one
+# proton and two neutrons) is often used for nuclear fusion. We use triton
```

Comment:
Isn't that [Tritium](https://en.wikipedia.org/wiki/Tritium)? @youkaichao and @ptillet :atom: 😄

Reply:
ChatGPT tells me that tritium is an atom and triton is its nucleus.

Reply:
Oh, right, wikipedia also says so:

> The [nucleus](https://en.wikipedia.org/wiki/Atomic_nucleus) of tritium (t, sometimes called a triton) contains one [proton](https://en.wikipedia.org/wiki/Proton) and two [neutrons](https://en.wikipedia.org/wiki/Neutron)

So it's actually `triton = tritium - electron`. Fun stuff.

Reply:
Let's make it this way: `tritium = triton  + electron`. Can't wait for someone to create a tool based on `electron` to use `triton`, and name it `tritium` 😄